Source/WebInspectorUI/UserInterface/Views/ContextMenuUtilities.js

```diff
@@ -178,24 +178,27 @@ WI.appendContextMenuItemsForSourceCode = function(contextMenu, sourceCodeOrLocat
 
     if (sourceCode instanceof WI.Resource && !sourceCode.localResourceOverride) {
         if (sourceCode.urlComponents.scheme !== "data") {
+
+            let isDisabled = !sourceCode.hasMetadata;
```

Comment:
i kinda feel like we shouldn't even show these contextmenu items if we don't have the data necessary to perform the action (e.g. that's normally what we do when the action relies on protocol changes), just like how we don't even show them for `data:` resources 🤔

Reply:
Ok. Not showing context menu items for resources without metadata.